### PR TITLE
No need to mention enwiki anymore

### PR DESCRIPTION
**Pull request by berndsi**

### CODE REVIEW

File: routes/mobileapplite1.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+ * @param {Object} params an Object with all the query parameters for the MW API
+ * @return {Promise} a promise resolving as the HTML element object
+ */
+function apiGet(domain, params) {
+
+    // get the page from the MW API
+    return preq.get({
+        uri: 'http://' + domain + '/w/api.php',
+        query: params
+    }).then(function(callRes) {
```
Comment (d00rman) [inline routes/mobileapplite1.js]: No need for this `then()` block, you can safely return what `preq.get()` returns and use the `callRes` object there. A `then()` block would be useful here only if you plan to somehow process the result before returning it.

File: routes/mobileapplite1.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    rmSelectorAll(doc, 'div.infobox');
+    rmSelectorAll(doc, 'div.hatnote');
+    rmSelectorAll(doc, 'div.metadata');
+    rmSelectorAll(doc, 'table'); // TODO: later we may want to transform some of the tables into a JSON structure
+    return doc.body.innerHTML;
+
+}
+
+
+function rmSelectorAll(doc, selector) {
```
Comment (d00rman) [inline routes/mobileapplite1.js]: The function needs to be declared before `runDomTransforms()` (where it is used)

File: routes/mobileapplite1.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            //res.status(200).type('json').end(apiRes);
+        });
+
+});
+
+
+/**
+ * Nuke stuff from the DOM we don't want.
+ */
+function runDomTransforms(text) {
```
Comment (d00rman) [inline routes/mobileapplite1.js]: This needs to be put in front of `router.get()` as it's used there.

**Comment by d00rman** [comment]
@berndsi  I think the mobile app routes do not belong in the template. It'd better if you start a new repo for your prototype (or simply rename your fork of this repo), and if/when you find bugs in the template, do PRs against this repo's master.

**Comment by berndsi** [comment]
@d00rman I agree. I only wanted to put the first commit into the pull request, not the second one. Still getting used to Github. Am more used to Gerrit ATM. It seems you cannot fork the same repo twice. This is why I added it there. Was much simpler.

**Pull request merged**